### ISSUE TITLE
Python DEBUG CONSOLE not refreshing

### ISSUE THREAD

**Issue by seg-sdanie** [microsoft/vscode#193075]
Type: <b>Bug</b>

During Python debugging in I observed, standing on a breakpoint, that the **DEBUG CONSOLE** is not properly refreshed. 

I enter a `print("Hello")` and neither the command entered nor the output prompt is shown in the **DEBUG CONSOLE** window.

Workaround: I click on a tab like **PORTS** and switch back to **DEBUG CONSOLE**  and the expected output is visible. 

VS Code version: Code 1.82.1 (6509174151d557a81c9d0b5f8a5a1e9274db5585, 2023-09-08T08:45:05.575Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (12 x 2592)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.76GB (22.21GB free)|
|Process Argv|--crash-reporter-id 846d1dc6-7e7a-4f2c-bafb-b45f8b666c5c|
|Screen Reader|no|
|VM|50%|
</details><details><summary>Extensions (44)</summary>

Extension|Author (truncated)|Version
---|---|---
bootstrap5-vscode|Anb|0.4.2
vscode-color|ans|0.4.5
vscode-log-viewer|ber|0.14.0
vscode-postgres|cko|1.4.3
doxdocgen|csc|1.4.0
vscode-postgresql-client2|cwe|6.7.2
vscode-empy|dir|0.1.1
prettier-vscode|esb|10.1.0
gc-excelviewer|Gra|4.2.58
todo-tree|Gru|0.0.226
vscode-drawio|hed|1.6.6
vscode-totalcommander|ipa|1.0.4
plantuml|jeb|2.17.5
better-cpp-syntax|jef|1.17.2
rainbow-csv|mec|3.7.0
perforce|mjc|4.15.7
vscode-language-pack-de|MS-|1.82.2023091309
isort|ms-|2023.10.1
python|ms-|2023.16.0
vscode-pylance|ms-|2023.9.10
jupyter|ms-|2023.8.1002501831
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.17
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.309.0
remote-ssh|ms-|0.106.4
remote-ssh-edit|ms-|0.86.0
remote-wsl|ms-|0.81.3
cmake-tools|ms-|1.15.31
cpptools|ms-|1.17.5
cpptools-extension-pack|ms-|1.3.0
hexeditor|ms-|1.9.12
powershell|ms-|2023.6.0
remote-explorer|ms-|0.4.1
sqlite-viewer|qwt|0.3.13
docxreader|Sha|1.1.3
markdown-preview-enhanced|shd|0.7.6
svg-preview|Sim|2.8.3
even-better-toml|tam|0.19.2
cmake|twx|0.0.17
vscode-icons|vsc|12.5.0
markdown-pdf|yza|1.5.0
markdown-all-in-one|yzh|3.5.1

(2 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30410667
py29gd2263cf:30792227
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
vscccc:30803845
3biah626:30602489
f6dab269:30613381
2i9eh265:30646982
showlangstatbar:30737416
962ge761:30835153
03d35959:30757346
pythonfmttext:30731395
fixshowwlkth:30771522
showindicator:30805244
pythongtdpath:30769146
i26e3531:30792625
pythonnosmt12:30797651
pythonidxptcf:30805731
pythonnoceb:30805159
dsvsc013:30795093
dsvsc014:30804076
diffeditorv2:30821572
dsvsc015:30829745

```

</details>

**Comment by gjsjohnmurray** [comment]
/duplicate, and will be fixed in upcoming 1.82.2

**Comment by roblourens** [comment]
https://github.com/microsoft/vscode/issues/192653
